wremnants/histselections.py

```diff
@@ -24,6 +24,15 @@ def fakeHistIsoRegionIntGen(h, scale=1.):
     print("Slicing")
     return h[{"iso" : 0, "qTgen" : s[::hist.sum]}]
 
+def fakeHistIsoRegionIntGen(h, scale=1.):
```

Comment:
Is this something I did for the low pileup or did you implement this? If it was me, I don't quite remember the use case, so probably better to remove it and restore it when/if it's needed.

Reply:
I didn't recall implementing this, will remove for now.